simpleLooping/listGreatestCommonDivisor.py

```diff
@@ -4,3 +4,33 @@
 # Ex:
 #   listGreatestCommonDivisor(-42, 402, -66, 354, 78, 426) # return 6
 #   listGreatestCommonDivisor(7883933, 91928167, 54151651) # return 1
+from typing import List
+
+def gcd(a: int, b: int) -> int:
```

Comment:
This is not a bad approach, but a function should follow the 3 principle:

1/ A function should do one thing.
2/ A function should do all of the thing.
3/ A function should do only that thing.

In this case, your funtion did not check for if those 2 variable are negative or something like that. So it is better to name them like

Option 1: gcdHelper - so that people know that this is a "helper function" not a fully/independanly function
Option 2: gcdTwoPositiveNumber - so that people know that this function only work if the input number are positive.

Reply:
By the way, if you decide to use this function, you should name the variable something more representative like num1/num2 or numA/numB.